nih-sparc/sparc-app
# Project details datasets tab #4e7p2n

# Description

The purpose of this PR is to add the datasets tab to the project detail page.

You'll need the latest changes from `sparc-api` running locally to test this locally.

## Type of change

# How Has This Been Tested?

- Go to a [Project page with datasets](http://localhost:3000/projects/1HHdZpD4JlOyDhD6rUvUp0)
- Click on a datasets name
- You should be on that dataset's detail page
- Go to a [Project page with no datasets](http://localhost:3000/projects/6F2XknXtQMQpsNKRuti8e3)

# Checklist:
